suspect/io/twix.py

```diff
@@ -382,44 +382,53 @@ def load_twix(filename):
 
 def anonymize_twix_header(header_string):
     """Removes the PHI from the supplied twix header and returns the sanitized version.
-
     This consists of:
     1) Replacing the patient id and name with strings of lower case x
     characters.
-    2) Replacing the patient birthday with 19700101
+    2) Replacing the patient birthday with 19900101
     3) Replacing the patient gender with the number 0
-    4) All references to the date and time of the exam have all alphanumeric
```

Comment:
Yeah, I keep wondering about having more control over the anonymisation, so that you could change only certain values or set them to replacement values, e.g. for changing the name/id. Not sure what a good interface would be though.